html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style></style>
+    <title>html-forms</title>
+    <style>
+      textarea {
+        width: 250px;
+        height: 150px;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label>Email</label>
+      <br />
+      <input type="email" name="email" required />
+      <br />
+      <label>Why is Coding Cool?</label>
```

Comment:
The label needs a `for` that matches the `id` of the `input` you want to associate it with